core/include/aes_gcm_dev.h

```diff
@@ -1,16 +1,16 @@
 #pragma once
 
-#include <squareup/subzero/internal.pb.h>
+/* This header is meant for the dev target, and should only be included in the
+ * dev implementation */
+
+// M_KeyID typedef for the dev target
+typedef enum {master_seed_encryption_key = 1,
+              pub_key_encryption_key = 2} M_KeyID;
 
-// M_KeyID is typedef'ed in codesafe header. If STDMARSHALTYPES_H is not
-// defined, we are in dev target
-#ifndef STDMARSHALTYPES_H
```

Comment:
This is a good change to separate this out. One less way that we can include the wrong thing if the `#defines` changes.

Reply:
Yeah, learned it hard way.